src/cljs/rems/applications.cljs

```diff
@@ -43,7 +43,7 @@
                                      :default-sort-column :created
                                      :default-sort-order :desc}]
         (when (roles/show-all-applications? (:roles identity))
-          [:<>
+          [:div.mt-3
```

Comment:
This will not fix any of the other tables, like in the screenshots the problem exists at least in the Catalogue page. Possibly every page that has search-component followed by the list-component. We should look for a solution that sets the margin to them and not to any wrapper div outside which is different for each page. And I think we have to check manually every table to see that the margin is correct and possible margin collapse is working correctly.

Reply:
Ok, what do you sugggest/.

Reply:
I think there is no avoiding of going through all the tables by hand and checking which change would fix them all. Certainly adding `.mt-3` sounds like wrong, it should perhaps work if you added that to the list component but if it doesn't then have to figure out why.